hamming/hamming_test.rb

```diff
@@ -1,11 +1,5 @@
 require 'minitest/autorun'
-begin
-  require_relative 'hamming'
-rescue LoadError => e
-  puts "\n\n#{e.backtrace.first} #{e.message}"
-  puts DATA.read
-  exit 1
-end
+require_relative 'hamming'
 
```

Comment:
Hamming gets generated from shared data. Take a look at the `example_gen.rb`. (I'm actually in the middle of refactoring this, so if you're OK with holding off until @kotp has a chance to review the generator PR (https://github.com/exercism/xruby/pull/134) then it might be easier to clean the new template up than the old `example_gen.rb` file.